api-scripts/feesfines/README.md

```diff
@@ -0,0 +1,22 @@
+## Workflow steps:
+
+##### api for GET_accounts.jmx
+1. GET /accounts?query=loanId==(<loanUUID> or <loanUUID>)&limit=2147483647
+
+
+
+## Modules required to be enabled as a pre-requisite to run JMeter script:
+##### Backend:
+- mod-authtoken-2.4.0
+- mod-inventory-storage:19.4.4
+- okapi:4.3.3
+##### Frontend:
+- folio_circulation-2.0.0
+
+- FOLIO release: HONEYSOUCKLE
```

Comment:
Nitpick: typo should be `Honeysuckle`

Reply:
Will resolve in scope of https://github.com/folio-org/perf-testing/pull/43#discussion_r551130416